ox_profile/core/metrics.py

```diff
@@ -51,11 +51,9 @@ def snap(self, frame):
 
         """
         stack = []
-        self.name = '{}({})'.format(frame.f_code.co_name,
-                                    frame.f_globals.get('__name__'))
+        self.name = '%s(%s)' % (frame.f_code.co_name, frame.f_globals.get('__name__'))
         while frame is not None:
-            formatted_frame = '{}({})'.format(frame.f_code.co_name,
-                                              frame.f_globals.get('__name__'))
+            formatted_frame = '%s(%s)' % (frame.f_code.co_name, frame.f_globals.get('__name__'))
```

Comment:
Same as previous comment: please fix to be less than 80 characters wide.

Reply:
Fixed